apps/platform/trpc/routers/convoRouter/convoRouter.ts

```diff
@@ -1610,7 +1622,10 @@ export const convoRouter = router({
             },
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 1618]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->get the participant ID here, and use that for the update later instead of searching for the public id<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/un/inbox/562?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->